go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stellar/soroban-tools
 
-go 1.18
+go 1.20
```

Comment:
What's the reason for this? This enforces a minimum Go version of 1.20 to compile the project. I think it should be 1.18 if we want to support the latest two releases.

Reply:
(I am sure that there will be patch releases for 1.19 and 1.18 containing the important security fixes)